Patch llama.py for internal build

cccclai opened the pull request:
Summary:
As title, minor changes so we can use buck to build..

```
buck run mode/dev-nosan //executorch/examples/qualcomm/oss_scripts/llama3_2:llama_qnn -- --compile_only --ptq 16a4w --checkpoint /home/chenlai/local/models/consolidated.00.pth --params /home/chenlai/local/models/params.json --tokenizer_model /home/chenlai/local/models/tokenizer.model --prompt "Once" -m SM8650 --model_size 1B --model_mode kv  2>&1 | tee static_llama.log
```

Differential Revision: D66947240

shewu-quic reviewed (approved):
LGTM.
Thanks for your patch.

cccclai committed:
Patch llama.py for internal build (#7239)

Summary:

As title, minor changes so we can use buck to build..

```
buck run mode/dev-nosan //executorch/examples/qualcomm/oss_scripts/llama3_2:llama_qnn -- --compile_only --ptq 16a4w --checkpoint /home/chenlai/local/models/consolidated.00.pth --params /home/chenlai/local/models/params.json --tokenizer_model /home/chenlai/local/models/tokenizer.model --prompt "Once" -m SM8650 --model_size 1B --model_mode kv  2>&1 | tee static_llama.log
```

Differential Revision: D66947240
